recipes/jkqtplotter/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "jkqtplotter" %}
+{% set version = "2019.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jkriege2/JKQtPlotter/archive/v{{ version }}.tar.gz
+  sha256: 94d39834c6f4e4afd4ad8511231a26d74fd88dc60d7b3ccd85b7ce00f32ff487
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - qt
```

Comment:
Is this needed on `build`?

Reply:
It's needed to build the library (it depends on `qt` headerfiles). I think that means it needs to be in `build`.

Reply:
`build` requirements are what are needed in the build machine. For eg: compilers, make, cmake, etc. `host` requirements are headers, libraries need to compile and link. For eg: if you are cross compiling from `linux` to `osx`, `build` are the build tools that should run in `linux`. `host` are things like `qt` headers and libraries built for `osx`.

Reply:
```suggestion
```

Reply:
I see, thanks for the explanation. I think it belongs into the `host` section, so I moved it there in 5469f8ecdc9a902c5becd3ddf808565660d5e7c4.